docs/redocly.yaml

```diff
@@ -0,0 +1,59 @@
+organization: Meshery
+
+apis:
+  sample@v1:
+    root: _data/swagger.yml
+extends:
+  - recommended
+rules:
+  no-unused-components: error
+theme:
+  openapi:
+    htmlTemplate: _layouts/redocly.html
+    sortTagsAlphabetically: true
+
+    # theme:
```

Comment:
@Freedisch is none of this configurable?

Reply:
No, It's not. By default, all these functionalities are not applicable